support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -76,13 +84,84 @@ const defaultContributeCopy = (
     <span className="gu-content__blurb-blurb-last-sentence"> Every contribution, however big or small, is so valuable for our future.</span>
   </span>);
 
+const articleCountContributeCopy = currency => (
+  <span>
+    Thank you for turning to the Guardian on so many occasions.
+    Show your support for our open, fiercely independent journalism today with a contribution of any size.
+    Every {currency}1 we receive is so valuable for our future.
+  </span>
+);
+
 const defaultHeaderCopyAndContributeCopy: CountryMetaData = {
   headerCopy: defaultHeaderCopy,
   contributeCopy: defaultContributeCopy,
 };
 
 // ----- Render ----- //
 
+type ArticleCountOptOut = {|
+    hasOptedOut: boolean;
+    onArticleCountOptOut: () => void;
+    onArticleCountOptIn: () => void;
+|}
+
+const ARTICLE_COUNT_OPT_OUT_COOKIE = {
+  name: 'gu_article_count_opt_out',
+  daysToLive: 90,
+};
+
+const DAILY_ARTICLE_COUNT_STORAGE_KEY = 'gu.history.dailyArticleCount';
+const WEEKLY_ARTICLE_COUNT_STORAGE_KEY = 'gu.history.weeklyArticleCount';
+const ARTICLES_THIS_WEEK_STORAGE_KEY = 'gu.history.articleCountsThisWeek';
+
+export const addArticleCountOptOutCookie = (): void =>
+  set(
+    ARTICLE_COUNT_OPT_OUT_COOKIE.name,
+    new Date().getTime().toString(),
+    ARTICLE_COUNT_OPT_OUT_COOKIE.daysToLive,
+  );
+
+export const removeArticleCountOptOutCookie = (): void =>
+  remove(ARTICLE_COUNT_OPT_OUT_COOKIE.name);
+
+export const removeArticleCountFromLocalStorage = (): void => {
+  window.localStorage.removeItem(DAILY_ARTICLE_COUNT_STORAGE_KEY);
+  window.localStorage.removeItem(WEEKLY_ARTICLE_COUNT_STORAGE_KEY);
+  window.localStorage.removeItem(ARTICLES_THIS_WEEK_STORAGE_KEY);
+};
```

Comment:
We don't need to do the `localstorage` stuff here as article count only exists in `localstorage` on dotcom. We will however need to do a small PR on `dcr`.
 
In dcr, when we check if a user has opted out, we will need to clear localstorage if they have: https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/components/App.tsx#L251

We actually already have this logic in `frontend`: https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/commercial/contributions-service.js#L98 so we're all good there!

Reply:
👍 I'll remove these lines